week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -12,37 +12,60 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). 
+     // b is not defined - replace x by b. 
+
   2. Remove the line that throws the error.
+     
   3. What is printed to the console?
-  4. How many times is "f1" called?
+  2, 6, 0, 2, 4, 6, 9, 13.    // @ mentor can you please help me understand why the output is 2, 6, 0, 4, 9, 6, 13, 8? 
+                              // you can see my reasoning in my notes below. Where am I wrong? 
+
```

Comment:
It should be: 2 6 4 9 6 13 8
You have an extra 0 after 6.

Reply:
Please get in touch with your mentor and ask for explaining it a live coding call, it'd be easier and faster to understand if you see it in live.

Reply:
Thanks for the feeback @zoltan-gal. I'll work on the exercises again after I'm done with this week's homework.